src/test/java/kitchenpos/application/MenuGroupServiceTest.java

```diff
@@ -0,0 +1,72 @@
+package kitchenpos.application;
+
+import kitchenpos.dao.MenuGroupDao;
+import kitchenpos.domain.MenuGroup;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.dao.DataIntegrityViolationException;
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+
+import static kitchenpos.TestFixtureFactory.createMenuGroup;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+@Sql("/truncate.sql")
+@SpringBootTest
+class MenuGroupServiceTest {
+    @Autowired
+    private MenuGroupService menuGroupService;
+
+    @Autowired
+    private MenuGroupDao menuGroupDao;
+
+    @Transactional
+    @DisplayName("새로운 MenuGroup을 추가할 수 있다.")
+    @Test
+    void createMenuGroupTest() {
+        MenuGroup menuGroup = createMenuGroup("회식세트메뉴");
+
+        MenuGroup savedMenuGroup = menuGroupService.create(menuGroup);
+
+        assertAll(() -> {
+            assertThat(savedMenuGroup).isNotNull();
+            assertThat(savedMenuGroup.getId()).isNotNegative();
+            assertThat(savedMenuGroup.getName()).isEqualTo(menuGroup.getName());
+        });
```

Comment:
```suggestion
        assertAll(
                () -> assertThat(savedMenuGroup).isNotNull(),
                () -> assertThat(savedMenuGroup.getId()).isNotNegative(),
                () -> assertThat(savedMenuGroup.getName()).isEqualTo(menuGroup.getName())
        );
```

기존 assert문의 첫 번째 테스트가 실패하면 다음 테스트를 진행하지 않는 문제를 해결하기 위해 assertAll을 사용하는데요!
즉, assertAll은 여러 assert 검증을 묶어서 테스트할 수 있게 해주는 역할을 합니다.
그렇다면 테스트를 위와 같이 작성해야 목적을 달성할 수 있을 것 같습니다~

Reply:
앗... 확인해보니 Executable을 가변 인자로 받는 군요!
기존에 작성한 코드는 전체 assertThat 구문들을 하나의 Excutable로 전달해서 assertAll이 지원하는 기능을 활용하지 못하고 있었네요 😢 
잘 모르고 사용하고 있었는데 알려주셔서 감사합니다!
수정했습니다~ 🙂